Add close_widget to hud_battle_log (#1758)
longturn/freeciv21: docs/Manuals/client-manual.rst

    Enables or Disables populating new turn information in a widget on the `Map View`_.

Show Detailed Combat Information
    Enables or Disables populating the :guilabel:`Battle Log` widget. When enabled you will see a widget

>     appear on the screen (typically in the upper left corner) after conflict occurs in your nation's vision.

Reviewer: ```suggestion
    appear on the screen (typically in the upper left corner) when a combat occurs in your nation's vision.
```
